### PR TITLE
fix: some style updates

### LINKED ISSUE #2345: Dark mode sidebar UI issues
Latest release - sidebar selected item is not readable

Screenshot 2023-05-12 at 15 52 49

### LINKED ISSUE #2350: Sidebar expand/collapse​ shortcut not working in desktop app
### What happened?

should work like it does on app.affine.pro with ctrl+'/'

### Version

Windows x64

### What browsers are you seeing the problem on?

Other

### Relevant log output

_No response_

### Anything else?

_No response_

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

### PR DESCRIPTION
**Pull request by pengx17**
fix #2350
fix #2345

### CODE REVIEW

**Comment by pengx17** [comment]
@Himself65 should be ready for review

File: apps/web/src/components/blocksuite/workspace-header/styles.css.ts [hunk 1]
```diff
@@ -7,12 +7,15 @@ export const headerContainer = style({
...
   // @ts-ignore
-  WebkitAppRegion: 'drag',
   zIndex: 'var(--affine-z-index-popover)',
   selectors: {
     '&[data-has-warning="true"]': {
       height: '96px',
     },
+    '&[data-sidebar-floating="false"]': {
+      // @ts-ignore
+      WebkitAppRegion: 'drag',
```
Comment (himself65) [inline apps/web/src/components/blocksuite/workspace-header/styles.css.ts:16]: remove `ts-ignore` and use `as ComplexStyleRule` instead

File: tests/parallels/open-affine.spec.ts [hunk 2]
```diff
@@ -26,33 +26,6 @@ test('Open last workspace when back to affine', async ({ page }) => {
...
 
-test('Open affine in first time after updated', async ({ page }) => {
-  await openHomePage(page);
-  const changeLogItem = page.locator('[data-testid=change-log]');
-  await expect(changeLogItem).toBeVisible();
-  const closeButton = page.locator('[data-testid=change-log-close-button]');
-  await closeButton.click();
-  await expect(changeLogItem).not.toBeVisible();
-  await page.goto('http://localhost:8080');
-  const currentChangeLogItem = page.locator('[data-testid=change-log]');
```
Comment (himself65) [inline tests/parallels/open-affine.spec.ts:37]: Im not sure if it is good to just remove these test cases.

Comment (pengx17) [reply tests/parallels/open-affine.spec.ts:37]: Change Log component is removed and will be integrated into the app updater button later

**Review by Himself65 — APPROVED** [review]

**Pull request merged**